src/mixins/statistics.ts

```diff
@@ -0,0 +1,46 @@
+import { BenefitLayout } from "@lib/types/benefits"
+import {
+  IRecorderEvent,
+  sendEventToRecorder,
+} from "@/lib/statistics-service/recorder.js"
+import {
+  IMatomoEvent,
+  sendEventToMatomo,
+} from "@/lib/statistics-service/matomo.js"
+import { BehaviourEventTypes } from "@lib/enums/behaviour-event-types.js"
+
+export default {
+  methods: {
+    sendEventsToRecorder: function (
+      droits: BenefitLayout[] = [],
+      droitId: string | undefined = undefined,
```

Comment:
Pourquoi ne pas en profiter pour uniformiser les termes pour qualifier une aide ?
```suggestion
      benefits: BenefitLayout[] = [],
      benefitId: string | undefined = undefined,
```

Reply:
Ok, je vais regarder pour améliorer dans ce sens, il y a déjà une guideline de discutée à ce sujet, c'est un peu confus encore